src/server/index.js

```diff
@@ -14,22 +17,46 @@ const dbConnection = require('./dbConnection')
 const passportStrategies = require('./passport/strategies')
 const passportStrategyUtils = require('./passport/strategyUtils')
 const LocalStrategy = require('passport-local').Strategy
+const path = require('path')
+const sessionStore = new MongoSessionStore({ mongooseConnection: dbConnection} )
 
-app.use(bodyParser.json())
-app.use(mongoSanitize())
-
-app.use(expressSession({ 
-	store: new MongoSessionStore({ mongooseConnection: dbConnection} ),
+const cookieSession = {
+	store: sessionStore,
 	secret: config.session.secret, 
 	resave: config.session.resave, 
 	saveUninitialized: config.session.saveUninitialized,
 	cookie: { 
 		secure: config.session.cookie.secure,
 		maxAge: config.session.cookie.maxAge
-	} }))
+	}
+}
+
+const websocketCookieSession = {
+	key: 'connect.sid',
```

Comment:
Maybe you should put `'connect.sid'` into config.json.

Reply:
Do we really need it to be configurable? I can't think of any case when it would be useful.
Albeit moving it to constant would be a good idea. I'll do it.